docs/Reference/Hooks.md

```diff
@@ -399,10 +399,17 @@ open connection to a database.
 
 The first argument is the Fastify instance, the second one the `done` callback.
 ```js
+// callback style
 fastify.addHook('onClose', (instance, done) => {
   // Some code
   done()
 })
+
+// or async/await style
+fastify.addHook('onClose', async (instance) => {
+  // Some async code
+  await closeDatabaseConnections()
+})
```

Comment:
Can you explain how this fits with the lead in sentence?

Reply:
Do you mean this line before the code block ?
> The first argument is the Fastify instance, the second one the `done` callback.

An alternative description could be:
> The hook function takes the Fastify instance as a first argument, and a `done` callback for synchronous hook functions.

Reply:
> Do you mean this line before the code block ?

Yes, that is the lead in sentence for the example.

> An alternative description could be:

That seems like a good change.

Reply:
Done in https://github.com/fastify/fastify/pull/3541/commits/09436e529f4643d8b38b9349161f496f2c7622ee.